packages/frontend/src/contexts/game/reducer/index.js

```diff
@@ -18,6 +18,12 @@ export const gameReducer = (state, action) => {
   }
 
   switch (action.type) {
+    case types.START_GAME: {
+      return {
+        ...state,
+        stage: GAME_STAGES.PLAYING,
+      };
+    }
```

Comment:
We don't as of now. Currently the only place the game stage gets changed is in the game reducer in the HANDLE_FLIPPED_PAIR action where it changes from playing to game over.

Since the stage: value is stored in the game context and the currentStep: value is in the form context, I thought that a reducer action would be needed so that we could call it based off of the form's currentStep.